src/components/timeline/Row.svelte

```diff
@@ -739,13 +853,15 @@
       on:contextMenu
       {selectedActivityDirectiveId}
       {selectedSpanId}
+      {selectedExternalEventId}
     >
-      {#if hasActivityLayer && activityOptions?.displayMode === 'grouped'}
+      <!--TODO: change make sure this works correctly-->
```

Comment:
Not sure this is working correctly for events

Reply:
Fixed in [fix filterByTime for events](https://github.com/NASA-AMMOS/aerie-ui/commit/9acea3e069630b3cf50db37bccfe26eb4a5a9dd0).